test/config/dbconfig.php

```diff
@@ -4,4 +4,3 @@
 $dbname='rating';
 $dbserver='postgres';
 $dbport='';
-?>
```

Comment:
did you mean to delete this?

Reply:
optional apparently https://www.php.net/manual/en/language.basic-syntax.instruction-separation.php#:~:text=The%20closing%20tag%20of%20a%20PHP%20block%20at%20the%20end,headers%20to%20the%20response%20later.

Reply:
yea generally good practice with php so you don't have newline errors with includes.